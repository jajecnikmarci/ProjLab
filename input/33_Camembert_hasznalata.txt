timercontrol enable
starttest
add room room1 3
add item room1 Camembert camembert
add person room1 Student student1
pickup student1
printstate room1
use student1 camembert
printstate room1
endtimer timer1
printstate room1
endtest