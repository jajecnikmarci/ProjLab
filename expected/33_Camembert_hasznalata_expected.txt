room1
	- capacity: 3
	- people: 
		student1
	- doors: 
	- items: 
	- poisonEffects: 
	- stunEffects: 
	- lastCleaning: null
room1
	- capacity: 3
	- people: 
		student1
	- doors: 
	- items: 
	- poisonEffects: 
		poison1
	- stunEffects: 
	- lastCleaning: null
